actors/builtin/miner/policy.go

```diff
@@ -273,25 +273,9 @@ func SectorDealsMax(size abi.SectorSize) uint64 {
 	return max64(256, uint64(size/DealLimitDenominator))
 }
 
-// Initial share of consensus fault penalty allocated as reward to the reporter.
-var consensusFaultReporterInitialShare = builtin.BigFrac{
-	Numerator:   big.NewInt(1), // PARAM_SPEC
-	Denominator: big.NewInt(1000),
-}
-
-// Per-epoch growth rate of the consensus fault reporter reward.
-// consensusFaultReporterInitialShare*consensusFaultReporterShareGrowthRate^consensusFaultGrowthDuration ~= consensusFaultReporterMaxShare
-// consensusFaultGrowthDuration = 500 epochs
-var consensusFaultReporterShareGrowthRate = builtin.BigFrac{
-	Numerator:   big.NewInt(100785473384), // PARAM_SPEC
-	Denominator: big.NewInt(100000000000),
-}
-
-// Maximum share of consensus fault penalty allocated as reporter reward.
-var consensusFaultMaxReporterShare = builtin.BigFrac{
-	Numerator:   big.NewInt(1), // PARAM_SPEC
-	Denominator: big.NewInt(20),
-}
+// Default share of block reward allocated as reward to the consensus fault reporter.
+// Applied as epochReward / consensusFaultReporterDefaultShare
+const consensusFaultReporterDefaultShare int64 = 4
```

Comment:
I opted to include the `ExpectedLeadersPerEpoch` const within the function instead of part of the const. Leaving `4` to represent this consensusFaultShare (and avoid conflating ideas of "block reward" with the "fault share"). Open to opinions, but this seems most orthogonal.